schemars/Cargo.toml

```diff
@@ -26,10 +26,12 @@ smallvec = { version = "1.0", optional = true }
 arrayvec = { version = "0.5", default-features = false, optional = true }
 url = { version = "2.0", default-features = false, optional = true }
 bytes = { version = "1.0", optional = true }
+sqlx = { version = "0.5", default-features = false, features=["json"], optional = true }
 
 [dev-dependencies]
 pretty_assertions = "0.6.1"
 trybuild = "1.0"
+sqlx = { version = "0.5", features=["runtime-async-std-native-tls", "json"] }
```

Comment:
This extra dep is because sqlx requires one of its async runtime/TLS features to be enabled to compile, but we don't want to enforce a particular runtime in normal builds.